frontend/src/metabase/actions/components/ActionViz/ActionParameterMapper.tsx

```diff
@@ -79,27 +81,45 @@ export const ActionParameterMappingForm = ({
 
   return (
     <div>
-      {actionParameters.map((actionParam: WritebackParameter) => (
-        <ParameterFormSection key={actionParam.id}>
-          <ParameterFormLabel>
-            {actionParam.name ?? actionParam.id}
-          </ParameterFormLabel>
-          <Select
-            value={currentMappings[getTargetKey(actionParam)] ?? null}
-            onChange={(e: SelectChangeEvent<string>) =>
-              handleParameterChange(e?.target?.value, actionParam.target)
-            }
-            options={[
-              { name: t`Ask the user`, value: null },
-              ...dashboardParameters.map(dashboardParam => ({
-                key: dashboardParam.id,
-                name: dashboardParam.name,
-                value: dashboardParam.id,
-              })),
-            ]}
-          />
-        </ParameterFormSection>
-      ))}
+      {actionParameters.map((actionParam: WritebackParameter) => {
+        const isHidden = isParameterHidden(action, actionParam);
+        const isRequired = isParameterRequired(action, actionParam);
+        const mappedValue = currentMappings[getTargetKey(actionParam)];
+        const showError = !mappedValue && isHidden && isRequired;
+        const name = actionParam.name ?? actionParam.id;
+
+        return (
+          <ParameterFormSection
+            key={actionParam.id}
+            data-testid={`parameter-form-section-${actionParam.id}`}
+          >
+            <ParameterFormLabel error={showError}>
+              <span>{`${name}${showError ? t`: is required` : ""}`}</span>
```

Comment:
could we use the same text we use for regular invalid form fields? like this 
<img width="718" alt="Screenshot 2023-06-07 at 19 26 04" src="https://github.com/metabase/metabase/assets/8542534/8e7a10d1-5279-4ae8-a6ed-33d56afe5dec">